swatch-contracts/src/main/java/com/redhat/swatch/contract/service/SubscriptionSyncService.java

```diff
@@ -75,7 +75,6 @@ public class SubscriptionSyncService {
   private final ProductDenylist productDenylist;
   private final EntityManager entityManager;
 
-  @Transactional
```

Comment:
Same as in https://github.com/RedHatInsights/rhsm-subscriptions/pull/3592/files#r1704022345.